### PR TITLE
Added weapon switching

### PR DESCRIPTION
**Pull request by Qmando**
Adds weapon switching

### CODE REVIEW

File: scripts/rocket.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+function Rocket(name, size, scene) {
```
Comment (eastebry) [inline scripts/rocket.js:1]: Figure out how to inherit this stuff!

File: scripts/gun.js [hunk 2]
```diff
@@ -55,6 +47,26 @@ function Gun(name, size, scene) {
 }
 
 
+Gun.prototype.deactivate = function() {
+    this.active = false;
+    $("#gun").remove();
```
Comment (eastebry) [inline scripts/gun.js:52]: This is going to cause errors in the moveGun function

File: scripts/rocket.js (hunk 1 above)
Comment (Qmando) [inline scripts/rocket.js:1]: Doing that now

File: scripts/gun.js (hunk 2 above)
Comment (Qmando) [inline scripts/gun.js:52]: roger

**Pull request closed without merging**